src/app/app.component.html

```diff
@@ -1,15 +1,51 @@
 <div class = "container">
-    <h3 class = "d-flex justify-content-center">File Sort App</h3>
-    <h5 class = "d-flex justify-content-center">Workspace</h5>
-
+  <div id="carouselExampleIndicators" class="carousel slide sizecontainer" data-bs-ride="carousel">
+    <div class="carousel-indicators">
+      <button type="button" data-bs-target="#carouselExampleIndicators" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
```

Comment:
why default value for aria-label?
use ngFor also)

Reply:
Right))I fixed.